php/php.editor/src/org/netbeans/modules/php/editor/completion/DocRenderer.java

```diff
@@ -575,6 +575,9 @@ private String composeDescription(String description, List<PHPDocBlock> comments
         }
 
         private String replaceInheritdocForDescription(String description, String parentDescription) {
```

Comment:
Missing `@NullAllowed` for `description` param.

Reply:
OT: And for `parentDescription` as well, could you please add it? Thanks.

Reply:
OT: Also, the whole method should be annotated by `@CheckForNull` (IIRC) since it can return `null` if I understand it correctly. Thanks.

Reply:
Resolving because of no change of this line in this MR (but still, would be nice to have it done).